styles.css

```diff
@@ -0,0 +1,1178 @@
+}
+
+.imagehtml {
+  position: absolute;
+  width: 50px;
+  height: 50px;
+  left: 12px;
+  border-radius: 8px;
+  top: calc(50% - 48px / 2 - 0.36px);
+  background-image: url(images/html5.svg);
+  background-repeat: no-repeat;
+}
+
+.javascript {
+  display: flex;
+  position: absolute;
+  justify-content: space-between;
+  width: 303px;
+  height: 59.72px;
+  left: 12px;
+  top: 48%;
+  background: #f7f7f9;
+  border-radius: 8px;
+}
+
+.titlehtml {
+  position: absolute;
+  height: 24px;
+  left: 76px;
+  right: 12px;
+  top: calc(50% - 24px / 2 - 0.36px);
+  font-family: 'Poppins', Times, serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #253858;
+}
+
+.imagejs {
+  position: absolute;
+  width: 50px;
+  height: 50px;
+  left: 12px;
+  border-radius: 8px;
+  top: calc(50% - 48px / 2 - 0.36px);
+  background-image: url(images/js.svg);
+  background-repeat: no-repeat;
+}
+
+.titlejs {
+  position: absolute;
+  height: 24px;
+  left: 76px;
+  right: 12px;
+  top: calc(50% - 24px / 2 - 0.36px);
+  font-family: 'Poppins', Times, serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #253858;
+}
+
+.frameworks {
+  display: flex;
+  position: absolute;
+  top: 70%;
+  left: 10px;
+  margin-left: 30px;
+  font-size: 30px;
+  color: #000;
+  text-decoration: none;
+}
+
+.frameworks-vector {
+  position: absolute;
+  left: 78%;
+  right: 33.33%;
+  width: 6px;
+  height: 12px;
+  top: 70%;
+}
+
+.separator_top {
+  position: absolute;
+  top: 75%;
+  height: 1px;
+  width: 327px;
+  margin-right: 20px;
+  margin-left: 20px;
+}
+
+.skills {
+  display: flex;
+  position: absolute;
+  height: 24px;
+  width: 237px;
+  font-size: 30px;
+  top: 78%;
+  margin-left: 30px;
+  color: #000;
+  text-decoration: none;
+}
+
+.skills-vector {
+  position: absolute;
+  left: 77%;
+  top: 78%;
+  width: 6px;
+  height: 12px;
+}
+
+.separator_bottom {
+  position: absolute;
+  top: 84%;
+  width: 327px;
+  margin-right: 20px;
+  margin-left: 20px;
+}
+
+.contactme {
+  position: absolute;
+  width: 375px;
+  height: 779px;
+  border-top-left-radius: 80px 80px;
+  left: 50px;
+  background: #6070ff;
+  top: 4060px;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+}
+
+.button_contactme {
+  font-family: 'Poppins', Tahoma, Geneva, Verdana, sans-serif;
+  font-style: normal;
+  font-weight: 550;
+  font-size: 20px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  position: absolute;
+  width: 140px;
+  height: 48px;
+  left: 24px;
+  top: 681px;
+  color: #6070ff;
+  text-decoration: none;
+  background: #fff;
+  border-radius: 8px;
+  border-color: #fff;
+  border: 2px solid #fff;
+}
+
+.button_contactme:hover {
+  color: #fff;
+  background: #6070ff;
+}
+
+#message {
+  position: absolute;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  width: 327px;
+  height: 170px;
+  padding: 15px 38px 15px 16px;
+  top: 488px;
+  margin-left: 24px;
+}
```

Comment:
- [ ] Kindly make sure you are using the correct `font family` in the **Textarea** field as the one provided by Figma design. I suggest you check correctly from the Figma design to make your design stick to the template.